filters.js

```diff
@@ -133,5 +133,26 @@ function isValidDate (date) {
   return date instanceof Date && !isNaN(date.getTime())
 }
 
+function multiply(v, arg) {
+  if (typeof(v) === "object" && typeof(arg) === "object") {
+    const numberKeys = filterNumericKeysFromObject(arg);
+    numberKeys.forEach(key => {
```

Comment:
add a check here for the other object to have the same numberic keys and show a warning if it is not the case